doc_source/chime-web-app.md

```diff
@@ -61,7 +61,7 @@ The Amazon Chime web application uses your default audio device\. If you use Moz
 
 ## Screen sharing options for the Amazon Chime web application<a name="web-app-sharing"></a>
 
-You can share your screen or an application window, if you are using the latest three versions of Mozilla Firefox or Google Chrome\. Screen share is available for Chrome version 72 and Firefox version 66 or later\.
+You can share your screen or an application window if you are using the latest three versions of Google Chrome\. Screen share is available for Chrome version 72 or later\.
```

Comment:
This might need updates in wording too? It's a little confusing to say that it's supported in the most recent 3 versions (Chrome is at version 91 now) and also 72 or later

Reply:
Hi! Thanks for the request. I'll check this with the manager in charge of meetings and get back to you. 

--Colin

Reply:
Word is that Firefox is only denied for internal Amazon users. So the doc is public facing and doesn't need to change.